features/step_definitions/email_steps.rb

```diff
@@ -71,6 +71,17 @@
   end
 end
 
+Then(/^#{capture_email} should contain escaped "(.*)"$/) do |email_ref, text| # rubocop:disable Cucumber/RegexStepName
+  escaped_text = Regexp.escape(text)
+
+  if email(email_ref).multipart?
+    email(email_ref).text_part.body.should =~ /#{escaped_text}/
+    email(email_ref).html_part.body.should =~ /#{escaped_text}/
+  else
+    email(email_ref).body.should =~ /#{escaped_text}/
+  end
+end
+
```

Comment:
Instead of escaping and matching with `should`, it would be nicer to use `expect` /`.to include`.
Following from that, `should contain exactly` might be a clearer title.